flight_planning/v1/telemetry.yaml

```diff
@@ -0,0 +1,220 @@
+info:
+  title: >-
+    Telemetry definitions used in flight planning interface.
+components:
+  schemas:
+    FlightTelemetry:
+      description: >-
+        When this information is present and the USS supports telemetry ingestion for a flight, the USS receiving this information should enqueue these telemetry reports to be delivered to the system, as if they were coming from the user, at (or soon after) the specified times.  If/when the associated flight is canceled or closed, any telemetry not yet delivered to the system should be dequeued and discarded.
+      type: object
+      properties:
+        states:
+          description: >-
+            The set of telemetry data that should be injected into the system (as if reported by the user or the user's system) at the appropriate times (and not before) for this flight.
+          type: array
```

Comment:
Should we have a min/max elements? I think generally good to upper bound the array size when possible/reasonable.

Reply:
Since this is data we're going to have appear to originate from a user of the USS's system, it seems like we wouldn't want to set min/max because that would preclude testing how the USS would respond if the user provided values that were outside those reasonable bounds in their telemetry.

Reply:
I wouldn't have made the assumption that these are data that are just sent raw to the USS... the API between the client and USS may very well not take ANY array of position data, let alone one with a limit.  I'm just thinking from a data security aspect... if we want more predicability in payloads or not.  If it isn't a security or resource concern, then fine not to have the upper limit. But, again, I wouldn't assume the API between a client and a USS at all... the USS under test needs to deal with this array as needed.